### ISSUE TITLE
code review

### ISSUE THREAD

**Issue by Junoth** [tomkelly110/Seedlings#2]
Hi,I've read your introduction and code basically.Actually,I really love your idea to use AutoML to finish simple classification and use transfer learning to finish complicate classification.I think your method is pretty smart because it uses suitable tools for different target.AutoML is a great tool.It can finish a series of task automatically like processing data,extracting features and training the model.It can adjust some parameters by itself.It can save you a lot of time then you can spend more efforts on complicate classification.That's great.As for transfer learning,I think it's a great choice to use VGG19 because the Convolution Layer of VGG19 is deep enough to extract enough features of plants.

I think,there are still some left to develop.Maybe I'm wrong and you can correct me lol.
1.The data of your train set is not enough.Maybe you can try to find more images on the website and add them to your train set.
2.I think the seven layer is not enough because if you have two plants that is hard to distinguish.Deep convolution layers can help you extract enough features.
3.It seems that you don't do enough pre-processing to images like crop,rotation and flip.That will make 
your dataset lack of space diversity.
4.I see the accuracy of your model access 100% and sometimes is 100%.Maybe you should consider about over-fitting,especially when your dataset is not enough.You should consider adding drop-out layer to your model.

**Comment by YanzuWuu** [comment]
Yeah,  thx for ur suggestions!  And firstly, we do think the dataset is too small, about 350 images for each label, but the project is put forward by kaggle, and the dataset they give us is that small. Perhaps we are gonna find some images from the internet. The second one, as u can see from the end of the readme, we are gonna give up the seven-layer one, due to the poor accuracy, and the vgg19 seems good (one of my partners is training it right now). Thirdly, it's a brilliant advice, and we r gonna make it in the sprint 3. Finally, the result which is approaching 100% is due to we tested the model by training set as we explained at the end of Readme. And we will update the vgg19 model as soon as possible.
